packages/backend/src/controllers/authentication.ts

```diff
@@ -164,18 +168,27 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
               ).href,
               passReqToCallback: true,
           },
-          async (req, issuer, profile, done) => {
+          async (
+              req: Express.Request,
+              accessToken: string,
+              refreshToken: string,
+              params: GoogleCallbackParameters,
+              profile: Profile,
+              done: VerifyCallback,
+          ) => {
               try {
+                  const issuer = 'https://accounts.google.com';
                   const { inviteCode } = req.session.oauth || {};
                   req.session.oauth = {};
-                  const [{ value: email }] = profile.emails;
+                  const [{ value: email }] = profile.emails || [];
```

Comment:
is the typing correct here? `email` could be undefined but not sure if typscript catches that